File: Level_4/1110.c
#include <stdio.h>

int main(void){
	int start, temp, second;
	int counter = 0;
	scanf("%d", &start);
	temp = start;
	do{
		if (start < 10 && counter == 0){
			temp = start % 10 * 11;
			counter ++;
		} else{
			second = temp % 10;
			temp = second + temp / 10;
			temp = second * 10 + temp % 10;
			counter ++;
		}
	} while (start != temp);
	printf("%d\n", counter);
	
	return 0;
}

File: Level_4/1110.py
start = int(input())
tmp, counter = start, 0

while True:
    if tmp < 10:
        tmp *= 11
        counter += 1
    elif tmp >= 10:
        tmp_1 = tmp % 10
        tmp = tmp // 10 + tmp_1
        tmp = tmp_1 * 10 + tmp % 10
        counter += 1
    if tmp == start : break
print(counter)